src/turnkeyml/llm/docs/llamacpp.md

```diff
@@ -0,0 +1,38 @@
+# LLAMA.CPP
+
+Run transformer models using a Llama.cpp binary and checkpoint. This model can then be used with chatting or benchmarks such as MMLU.
+
+## Prerequisites
+
+This flow has been verified with a generic Llama.cpp model.
+
+### Set up Environment (Assumes TurnkeyML is already installed)
+
+Build or obtain the Llama.cpp model and desired checkpoint.
+For example (see the [llama.cpp](https://github.com/ggerganov/llama.cpp/blob/master/docs/build.md
+) source for more details):
```

Comment:
On Windows, I needed to be on the Administrator prompt for this to work. It is worth noting here in the Windows instructions section.

Reply:
Done